File: 04447.cpp
#include <iostream>
#include <cstdio>
#include <string>
using namespace std;

int main() {
	int n; cin >> n;
	getchar();

	for (int i = 0; i < n; i++) {
		string s; getline(cin, s);

		int g = 0, b = 0;
		for (int j = 0; j < s.length(); j++) {
			if (s[j] == 'g' || s[j] == 'G') g++;
			else if (s[j] == 'b' || s[j] == 'B') b++;
		}

		if (g == b)
			cout << s << " is NEUTRAL\n";
		else if (g > b)
			cout << s << " is GOOD\n";
		else
			cout << s << " is A BADDY\n";
	}
}

File: 04447.py
for i in range(int(input())):
    s = input()

    g = 0; b = 0
    for j in range(len(s)):
        if s[j] == 'g' or s[j] == 'G':
            g += 1
        elif s[j] == 'b' or s[j] == 'B':
            b += 1

    if g == b:
        print(s, "is NEUTRAL")
    elif g > b:
        print(s, "is GOOD")
    else:
        print(s, "is A BADDY")
